packages/math/base-elements.lua

```diff
@@ -1385,40 +1435,43 @@ local function _r (number)
 end
 
 function elements.sqrt:output (x, y, line)
-   -- HACK FIXME:
+   -- HACK:
    -- OpenType might say we need to assemble the radical sign from parts.
    -- Frankly, it's much easier to just draw it as a graphic :-)
    -- Hence, here we use a PDF graphic operators to draw a nice radical sign.
    -- Some values here are ad hoc, but they look good.
    local h = self.height:tonumber()
    local d = self.depth:tonumber()
-   local sw = self.symbolWidth
-   local dh = h - self.symbolHeight
+   local s0 = scaleWidth(self.offsetX, line):tonumber()
+   local sw = scaleWidth(self.symbolWidth, line):tonumber()
+   local dsh = h - self.symbolShortHeight:tonumber()
+   local dsd = self.symbolDepth:tonumber()
    local symbol = {
-      _r(self.radicalRuleThickness),
-      "w", -- line width
-      2,
-      "j", -- round line joins
-      _r(sw),
+      _r(self.radicalRuleThickness), "w", -- line width
+      2, "j", -- round line joins
+      _r(sw + s0),
       _r(self.extraAscender),
       "m",
-      _r(sw * 0.4),
-      _r(h + d),
+      _r(s0 + sw * 0.90),
+      _r(self.extraAscender),
+      "l",
+      _r(s0 + sw * 0.4),
+      _r(h + d + dsd),
       "l",
-      _r(sw * 0.15),
-      _r(dh),
+      _r(s0 + sw * 0.2),
+      _r(dsh),
       "l",
-      0,
-      _r(dh + 0.5),
+      s0 + sw * 0.1,
+      _r(dsh + 0.5),
       "l",
       "S",
    }
    local svg = table.concat(symbol, " ")
    SILE.outputter:drawSVG(svg, x, y, sw, h, 1)
```

Comment:
And we must `scaleWidth(x, line)` here (or, e.g. stretching in inline text goes badly) - Also found during harness tests with a bunch of formulas.

Reply:
Should this be addressed here or promoted/postponed to a separate issue?

Reply:
I'd should have tested posted the fixes late yesterday, if my neighbor didn't have a water leak issue in her house and required our urgen help ;) -- I'll check my branch later today.

Reply:
Done, commit amended